primer-io/primer-sdk-ios
# CHKT-152 project improvements

# What this PR does

- Completely reviews the Example app
  - Adds support to [tuist.io ](https://tuist.io)
  - Removes completely all references to .xcworkspace and .pbxproj
  - Reviews main app target, Test, and UI Test target
  - UI Bugfixes
  - Project layout cleanup
  - Adds the Primer Icon to our Example App
- Reviews .gitignore
- Creates a small shell script to run Tuist and install cocoapods depending on whether we are on CI
- Removes the Pods folder without compromising our day-to-day work
- Reviews `Dangerfile.swift`
- Cleanup repository

# How to test

- Make sure you have Tuist installed on your mac
  - `curl -Ls https://install.tuist.io | bash`
- move to the root folder of this repository
- launch `sh ./Example/tuist-generate.sh`
